test/test.H

```diff
@@ -14,6 +14,21 @@
 #include <hobbes/util/str.H>
 #include <hobbes/util/stream.H>
 
+#include <unistd.h> // readlink
+#include <libgen.h> // direname, basename
+#include <linux/limits.h> // PATH_MAX
+
+template<class Fn>
+auto execPath(Fn && fn) -> void {
+  std::string r(PATH_MAX, 0);
+  auto count = ::readlink("/proc/self/exe", const_cast<char*>(r.data()), r.size());
```

Comment:
This probably breaks darwin, but I have no Mac to test this on. Will fix once we've figured out the issue with brew and mac osx builds